po/ru.po

```diff
@@ -118,24 +118,22 @@ msgstr "эта помощь"
 
 #: newsboat.cpp:121
 msgid "Files:"
-msgstr "Файлы"
+msgstr "Файлы:"
 
 #. / This is printed out by --help before the path to the config file
 #: newsboat.cpp:123
-#, fuzzy
 msgid "configuration"
-msgstr "Загружаю настройки..."
+msgstr "настройки"
 
 #. / This is printed out by --help before the path to the urls file
 #: newsboat.cpp:125
-#, fuzzy
 msgid "feed URLs"
-msgstr "Источники"
+msgstr "адреса лент"
```

Comment:
I'm assuming that `лент` means something like `file`.
Is it intentional that `"feed URLs"` and `"кэш лент"` include that word but `"настройки"` does not?

---
I expect those 3 strings are printed together as follows:
```
Файлы:
        - настройки:    /home/dennis/.newsboat/config
        - адреса лент:  /home/dennis/.newsboat/urls
        - кэш лент:     /home/dennis/.newsboat/cache.db
```

Reply:
"Лента" means "feed", so "адреса лент" is "feeds' addresses", and "кэш лент" is "feeds cache". I didn't use "URLs" in the translation because English abbreviations always look a bit jarring next to Cyrillic text, and even more so when pluralized (something). ; to get around that, I rephrased the

Reply:
Makes sense, thanks for the explanation!